lib/Position/WeatherParserUtils.js

```diff
@@ -0,0 +1,78 @@
+
+
+const wind_multiplier = 0.44704;
+const rain_multiplier = 0.254;
+const weatherRegEx = new RegExp(/([cSgtrpPlLs#]\d{3}|t-\d{2}|h\d{2}|b\d{5}|s\.\d{2}|s\d\.\d)/g);
+
+const weatherMap = {
+    'g': {
+        keyName: 'wind_gust',
+        val: x => { return Number(x) * wind_multiplier; }
+    },
+    'c': {
+        keyName: 'wind_direction',
```

Comment:
I think we should add some information about unit for each field, I think simple comments here would do